tesseract_ros/tesseract_rosutils/src/conversions.cpp

```diff
@@ -68,6 +70,68 @@ tesseract_msgs::ProcessPlanPath toProcessPlanPath(const tesseract_common::JointT
   return path;
 }
 
+bool toJointTrajectory(trajectory_msgs::JointTrajectory& joint_trajectory,
+                       const tesseract_planning::Instruction& instruction)
+{
+  if (tesseract_planning::isCompositeInstruction(instruction))
+  {
+    const auto* ci = instruction.cast_const<tesseract_planning::CompositeInstruction>();
+    auto fi = tesseract_planning::flatten(*ci, tesseract_planning::moveFilter);
+
+    double t = 0;
+    if (!joint_trajectory.points.empty())
+      t = joint_trajectory.points.back().time_from_start.toSec();
+
+    double last_time = 0;
+    double current_time = 0;
+    for (const auto& i : fi)
+    {
+      if (tesseract_planning::isMoveInstruction(i))
+      {
+        auto mi = i.get().cast_const<tesseract_planning::MoveInstruction>();
+
+        if (isStateWaypoint(mi->getWaypoint()))
```

Comment:
could use `const Eigen::VectorXd& getJointPosition(const Waypoint& waypoint);` to make this a bit more future proof.

Reply:
I was not sure how best to handle the other waypoints which do not have time. I would need to call getJointPosition and also create getJointNames and getTimeFromStart, but was not sure if that would be to messy. Any ides?

Reply:
I wouldn't worry about it for now. Practically speaking, it probably won't come up since I think all of our processes output a move instructions with state waypoints.

At some point we might want to think about the possibility of visualizing plan instructions too though.